Homework: bus tickets

HKV203 opened the pull request.

Commit:
Ticket validation and reading from files

Commit:
The validation and parser logic are moved to separate classes

VolgaDaniluk commented on src/main/java/com/danliuk/logic/Parsing.java:
```diff
@@ -0,0 +1,49 @@
+public class Parsing {
+
+    public static void parseByConsole() throws JsonProcessingException{
+        Validation val = new Validation();
```
Variable names should be short yet meaningful

VolgaDaniluk commented on src/main/java/com/danliuk/logic/Validation.java:
```diff
@@ -0,0 +1,125 @@
+    private int priceExceptions = 0;
+    private int ticketTypeExceptions = 0;
+
+    public void validate(BusTicket busTicket) {
```
the approach with the exception throwing and catching not a good one

VolgaDaniluk commented on src/main/java/com/danliuk/Main.java:
```diff
@@ -1,29 +1,29 @@
+            System.out.println("1. From console");
+            System.out.println("2. From file");
+            option = scanner.nextInt();
+        } while (option != 1 && option != 2);
```
are you sure that the condition is written correctly?

Commit:
1. Validation class variable name is changed
2. Validation approach with throwing and catching exceptions is replaced with a new one
3. The scanner.nextInt() is replaced with scanner.nextLine() so the program won't crash when other symbols are inputted

Commit:
Exceptions are ignored

HKV203 committed:
Delete src/main/java/com/danliuk/exceptions directory
